sync/tc.py

```diff
@@ -354,10 +388,21 @@ def is_status_fn(statuses):
 def lookup_index(index_name):
     if index_name is None:
         return None
-    idx_url = INDEX_BASE + "task/" + index_name
-    r = requests.get(idx_url)
-    idx = r.json()
-    task_id = idx.get("taskId")
+    error = None
+    for base in [INDEX_BASE, OLD_INDEX_BASE]:
+        idx_url = INDEX_BASE + "task/" + index_name
+        resp = requests.get(idx_url)
+        try:
+            resp.raise_for_status()
+        except requests.HTTPError as e:
+            error = e
+            continue
+        else:
+            error = None
+        idx = resp.json()
+        task_id = idx.get("taskId")
```

Comment:
Shouldn't these two lines be in the else block? Currently it looks like if it passes on the first try, it'll then try with the second index base, notice the error and then raise it further down in the `if error:` block.

Reply:
Good catch, thanks!